src/slurm/Makefile.am

```diff
@@ -0,0 +1,23 @@
+
+plugindir = $(libdir)/slurm
```

Comment:
Hm - I'm not sure I understand the question.  The SLURM plugin directory - for the SLURM RPMs - is `/usr/lib64/slurm`.

Are you suggesting that we make the SLURM plugin directory path a `configure.ac` option?